doc/howtos/rdtraining/setup.rst

```diff
@@ -0,0 +1,408 @@
+
+.. code-block:: console
+
+    $ sudo apt install postgresql postgresql-client
+    $ sudo -u postgres createuser -s $USER
+
+
+Some sql useful commands :
+
+.. code-block:: console
+
+    $ createdb $DB_NAME #Create a database
+    $ dropdb $DB_NAME #Drop a database
+
+    $ psql $DB_NAME #Connect into a database
+        \l #List all the available databases
+        \dt #List all the table of the $DB_NAME database
+        \d $TABLE_NAME #Show the structure of the table $TABLE_NAME
+        \q #Quit the psql environment (ctrl + d)
+
+Run the server
+==============
+
+Running odoo-bin
+----------------
+
+Once all dependencies are set up, Odoo can be launched by running odoo-bin, the command-line interface of the server.
+
+.. code-block:: console
+
+    $ cd src/odoo/
+    $ ./odoo-bin --addons-path="addons/,../enterprise/" -d rd-demo
+
+There are multiples :ref:`command-line arguments <reference/cmdline/server>` that you can use to
+configure the server. In this training, you will only need some of them.
+
+* `-d <database>`
+  The database that is going to be used.
+
+* `--addons-path <directories>`
+  Comma-separated list of directories in which modules are stored. These directories are scanned for
+  modules.
+
+* `--limit-time-cpu <limit>`
+  Prevents the worker from using more than <limit> CPU seconds for each request.
+
+* `--limit-time-real <limit>`
+  Prevents the worker from taking longer than <limit> seconds to process a request.
+
+The last two are going to be used to avoid killing the worker when debugging the source code.
+
+.. tip:: If when you start the server you have an error like `AttributeError: module '$MODULE_NAME' has no attribute '$ATTRIBUTE'`
+
+         You may need to re-install the module with `$ pip install --upgrade --force-reinstall $MODULE_NAME`
+
+         If this error occurs with more than one module you may need to re-install all the
+         requirements with `$ pip3 install --upgrade --force-reinstall -r requirements.txt`
+
+         You can also remove the python cache to solve the issue
+
+         .. code-block:: console
+
+            $ cd /home/$USER/.local/lib/python3.8/site-packages/
+            $ find -name '*.pyc' -type f -delete
+
+
+Log in to odoo
+--------------
+
+Open `http://localhost:8069/` at your browser. We recommend you to use: Firefox, Chrome
+(Chromium the open source equivalent) or any other browser with development tools.
+
+To log in as the administrator user, you can use the following credentials : email = `admin` ;
+password = `admin`.
+
+The developer mode
+==================
+
+The Developer or Debug Mode gives you access to extra and advanced tools.
+
+This will be useful during the whole training, for the rest of the training we will always assume
+that the user is in developer mode.
+
+To activate the developer or debug mode you can follow the steps `here`_.
+
+.. _here: https://www.odoo.com/documentation/user/13.0/general/developer_mode/activate.html
```

Comment:
Ideally it would be good to cross-reference this with `:ref:`. In the meantime, we can use:

```suggestion
https://www.odoo.com/documentation/user/general/developer_mode/activate.html
```
which redirects to the latest version